doc/rtd_environment.yml

```diff
@@ -2,7 +2,7 @@ channels:
   - conda-forge
   - defaults
 dependencies:
-  - python=3.8
+  - python=3.9
```

Comment:
Do we still need this @marscher ?

Reply:
I think this was important for Sphinx to find the docstrings of dataclass members. But I'm not sure right now. Does it hurt to use a recent version here?

Reply:
no, just was curious about what features 3.9 makes available here 👍